src/SUMMARY.md

```diff
@@ -67,6 +67,7 @@
 * [Python](languages/python.md)
 * [Ruby](languages/ruby.md)
 * [Go](languages/go.md)
+* [C# / .NET Core](languages/dotnet.md)
```

Comment:
In the main doc you only reference this runtime as `.NET Core`, and this is the only place where you use `C# /` before. I'm not very familiar, but should this sidebar entry also then just be `.NET Core`?

Reply:
because the menu item is called `Languages` but .NET is not a language but a framework.

under the frameworks menu we have exclusively php frameworks, such as drupal, wordpress or symfony, so it would not fit there.